torch/distributed/fsdp/_composable_utils.py

```diff
@@ -0,0 +1,94 @@
+"""
```

Comment:
curious about the naming of this file. This seems to be used by both composable and wrapper features. Any reason we only name this file with a composable prefix?

Reply:
Good point! This name does not make sense. Let me rename it to `traversal_utils`.